client/src/EstimatesComparison/scheme.js

```diff
@@ -501,12 +501,13 @@ export class EstimatesExplorer extends AbstractExplorerScheme {
       };
     }
     if (type === "column_header_click") {
-      const { is_descending, sort_col } = state;
-      const clicked_col = payload;
+      const { sort_col } = state;
+      const clicked_col = payload.col_key;
+      const direction = payload.direction;
       const mods =
         clicked_col === sort_col
-          ? { is_descending: !is_descending }
-          : { is_descending: true, sort_col: clicked_col };
+          ? { is_descending: direction === "DESC" }
```

Comment:
(well, if we were using `is_descending` I guess there could be a meaningful third option, which is when no sorting is applied, but that may be more useful as a separate prop for `is_sorting`, likely calculated by the consumer from something like `sort_col`, rather than as a special meaning for `is_descending === null`)